README.md

```diff
@@ -0,0 +1,8 @@
+# Pizza Roulette 🍕🍕🍕
+Sometime's it's really tough to choose a pizza restaurant because of the sheer amount that exists. 
+Pizza Roulette aims to fix this by first randomly selecting 20 of the top pizza restaurants in your location. 
+
+Then, the app will randomly select eight from the list of top 20 and place it on a spinner wheel and choose a restaurant for you. 
```

Comment:
grammer: "randomly select" doesn't match tense in the previous sentence "first randomly selecting"